spec/controllers/reviews_controller_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
+
+describe ReviewsController do 
+
+  describe "POST create" do 
+    let(:video) { Fabricate(:video) }
+    context "with authenticated users" do 
+      let(:current_user) { Fabricate(:user) }
+      before { session[:user_id] = current_user.id}
+      context "with valid input" do 
+        before do 
+          post :create, review: Fabricate.attributes_for(:review), :video_id => video.id
+        end
+
+        it "creates the review" do 
+          expect(Review.count).to eq(1)
+        end
+
+        it "creates a review associated with the video" do 
+          expect(Review.first.video).to eq(video)
+        end
+
+        it "creates a review associated with current user" do
+          expect(Review.last.user).to eq(current_user)
+        end
+
+        it "redirects to the video show page" do 
+          expect(response).to redirect_to video_path(video)
+        end
+        it "sets the notice" do 
+          expect(flash[:success]).not_to be_blank
```

Comment:
I would recommend using the happy path (positive matcher) whenever possible. You can use `to be_present`.